Documentation/help_files/openemr_installation_help.php

```diff
@@ -0,0 +1,589 @@
+<?php
+/**
+ * Access Control List Help.
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author Ranganath Pathak <pathak@scrs1.org>
+ * @copyright Copyright (c) 2019 Ranganath Pathak <pathak@scrs1.org>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+require_once("../..//library/htmlspecialchars.inc.php");
```

Comment:
Should we even be trying to translate this stuff since it is looked at before database exists? I am guessing all the xlt() will throw a bunch of warnings into the php error log.

Reply:
Before do anything big (ie. remove all the xlt calls), lets figure out best path first.